docs/3_static.md

```diff
@@ -0,0 +1,178 @@
+
+> **Managed Resource**: an instance of a cluster-scoped CRD that represents an
+> external unit of infrastructure. The fields of a managed resource CRD map
+> 1-to-1 with the fields exposed by the provider's API, and creation of a
+> managed resource result in immediate creation of external unit. The CRDs that
+> represent managed resources on a provider are installed with it.
+
+The fields available on the `CloudMemorystoreInstance` CRD match the ones
+exposed by GCP, so you can configure it however you see fit.
+
+Create a file named `cloudmemorystore.yaml` with the following content:
+
+```yaml
+apiVersion: cache.gcp.crossplane.io/v1beta1
+kind: CloudMemorystoreInstance
+metadata:
+  name: example-cloudmemorystore-instance
+spec:
+  providerRef:
+    name: gcp-provider
+  writeConnectionSecretToRef:
+    name: example-cloudmemorystore-connection-details
+    namespace: crossplane-system
+  reclaimPolicy: Delete
+  forProvider:
+    tier: STANDARD_HA
+    region: us-west2
+    memorySizeGb: 1
+```
+
+> *Note: there is no namespace defined on our configuration for the
+> `CloudMemorystoreInstance` above because it is
+> [cluster-scoped](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition).*
+
+Now, create a `CloudMemorystoreInstance` in your cluster with the following
+command:
+
+```
+kubectl apply -f cloudmemorystore.yaml
+```
+
+The GCP provider controllers will see the creation of this
+`CloudMemorystoreInstance` and subsequently create it on GCP. You can log in to
+the GCP console to view the the status of the resource, but Crossplane will also
+propagate the status back to the `CloudMemorystore` object itself. This allows
+you to manage your infrastructure without ever leaving `kubectl`.
+
+```
+kubectl describe -f cloudmemorystore.yaml
+```
+
+[TODO INSERT OUTPUT]
+
+When the resource is done provisioning on GCP, you should see the status turn to
+`Ready`, and a corresponding event will be emitted. At this point, Crossplane
+will create a `Secret` that contains any connection information for the external
+resource. The `Secret` is created in the location we specified in our
+configuration:
+
+```yaml
+writeConnectionSecretToRef:
+    name: example-cloudmemorystore-connection-details
+    namespace: crossplane-system
+```
+
+Take a look at the contents of the `Secret`:
+
+```
+kubectl -n crossplane-system describe secret example-cloudmemorystore-connection-details
+```
+
+You will also see that the `CloudMemorystoreInstance` resource is still
+reporting `Unbound`. This is because we have not *claimed* it for usage yet.
+
+Crossplane follows a similar pattern to [Kubernetes persistent
+volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/). When
+you statically provision a resource in Crossplane, the external resource is also
+created. However, when you want to use a resource, you create an abstract
```

Comment:
Nit: I'd prefer to avoid discussing claims as being (generically) "abstract" and instead focus on their alignment within the separation of concerns Crossplane provides. So perhaps something like "application focused claim" rather than "abstract claim".